pypesto/objective/__init__.py

```diff
@@ -7,10 +7,15 @@
 from .objective import Objective
 from .amici_objective import AmiciObjective
 from .aggregated import AggregatedObjective
-from .petab_import import PetabImporter
 from .options import ObjectiveOptions
 from .util import res_to_chi2, sres_to_schi2
 
+# PEtab is an optional dependency
+try:
+    from .petab_import import PetabImporter
+except ModuleNotFoundError:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [undefined name 'ModuleNotFoundError'](https://app.codacy.com/app/dweindl/pyPESTO/pullRequest?prid=4686900)

Reply:
duh. ModuleNotFoundError was introduced in python 3. so, seems like codacy messes up? Either change to `ImportError`, or try adding a .codacy.yml file having
```yml
engines:
  pylint:
    enabled: true
    python_version: 3
```

Reply:
Thanks!